model_config/bidaf.json

```diff
@@ -16,14 +16,14 @@
       }
     }
   },
-  "train_data_path": "/squad/train-v1.1.json",
-  "validation_data_path": "/squad/dev-v1.1.json",
+  "train_data_path": "https://s3-us-west-2.amazonaws.com/allennlp/datasets/mc/squad-train-v1.1.json",
```

Comment:
I think it'd be nicer to store things on S3 like `/datasets/mc/squad/[file]`, instead of `/datasets/mc/squad-[file]`, as the latter could get pretty cluttered as we add more machine comprehension datasets.

Reply:
@matt-gardner I'm going to put them in `/datasets/squad` instead.  I think this is better than what I had originally and will be easier to manage than organizing datasets by task additionally.

Reply:
@matt-gardner I'm going to keep the dataset name on the file in addition.  Although it's redundant, it makes it easier to understand downloaded files.

Reply:
Sounds good to me.